src/core/a-mixin.js

```diff
@@ -1,38 +1,72 @@
 /* global HTMLElement */
-var AComponents = require('./component').components;
 var ANode = require('./a-node');
 var registerElement = require('./a-register-element').registerElement;
+var components = require('./component').components;
 
+/**
+ * @member {object} attrsCache - Cache of pre parsed component attributes
+ */
 module.exports = registerElement('a-mixin', {
   prototype: Object.create(
     ANode.prototype,
     {
+      createdCallback: {
+        value: function () {
+          this.attrsCache = {};
```

Comment:
mixins will work with anything if you use ```mixinEl.getAttribute('xxx')```. The cache is just to preparse the component backed attributes to minimize parsing passes.